core/drand_control.go

```diff
@@ -638,3 +635,13 @@ func (d *Drand) pushDKGInfo(to []*key.Node, group *key.Group, secret []byte, tim
 	cancel()
 	return nil
 }
+
+func getNonce(g *key.Group) []byte {
```

Comment:
Jst highlighting an important point of this PR: I'm using the timestamp of the group as a nonce.